docs/pages/router/reference/troubleshooting.mdx

```diff
@@ -5,6 +5,16 @@ description: Fixing common issues with Expo Router setup.
 
 import { Terminal } from '~/ui/components/Snippet';
 
+## Missing files / source maps in React Native DevTools
+
+This can happen if your Chrome DevTools has exclusions within its ignore list. To fix the issue
```

Comment:
```suggestion
This can happen if your Chrome DevTools has exclusions within its ignore list. To fix the issue, use [React Native DevTools](https://reactnative.dev/docs/react-native-devtools):
```
- missing colon?
- what does exclusion mean here?
- add missing context about using React Native DevTools and a link to the tool page